pkg/cli/cmd/radinit/aws.go

```diff
@@ -29,7 +29,10 @@ import (
 
 const (
 	selectAWSRegionPrompt                 = "Select the region you would like to deploy AWS resources to:"
+	selectAwsCredentialKindPrompt         = "Select a credential kind for the AWS credential:"
 	enterAWSIAMAcessKeyIDPrompt           = "Enter the IAM access key id:"
+	enterAWSRoleARNPrompt                 = "Enter the role ARN:"
+	enterAWSRoleARNPlaceholder            = "Enter IAM role ARN..."
```

Comment:
Here you added `AWS` instead of `Aws`.

Reply:
I see we are using mix of these 2 formats, mostly i see `Aws` in ucp code and `AWS` on the cli side. I have updated it to `AWS` format

Reply:
updated it